### ISSUE TITLE
Add polyfills for PHPUnit 8

### ISSUE THREAD

**Issue by filips123** [PHPUnitGoodPractices/polyfill#6]
PHPUnit 8 has the return type declaration which is not compatible with older PHP and PHPUnit versions. There should be some polyfills for this.

See sebastianbergmann/phpunit#3543.

**Comment by keradus** [comment]
polyfill will not handle change of method prototype - eg explicit return typehint.

polyfill may add methods introduced in PHPUnit 8 to lower versions of PHPUnit.

**Comment by filips123** [comment]
Is it possible to also overwrite methods?

**Comment by keradus** [comment]
yes, see https://github.com/PHPUnitGoodPractices/polyfill/blob/master/src/PolyfillTrait7.php#L21 for example
we always override, and then conditionally do polyfilling or pass to parent method, if it exists

of course, method header has to be compatible
